source/hardware/controllers/native_instruments_traktor_kontrol_s2_mk3.rst

```diff
@@ -123,8 +122,14 @@ Mixxx mapping
 | (29) Headphone   | Adjusts the headphone output gain                                                                                           |                                                     |
 | gain knob        |                                                                                                                             |                                                     |
 +------------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------+
-| (30) Effect      | Enable or disable effect units for both decks                                                                               |                                                     |
+| 30. Effect       | | Enable effect from the quick effect presets list on both decks. 8 first effects from the list can be selected.            | Enable effect on selected deck                      |
```

Comment:
What's the timeout to make the "second" press a first press?

Reply:
I get what you want to illustrate with that table, but it's not clear if I can press 1, then 2 to get effect 6?

What do you think about this
```
1st press (2nd press)

1 (5)  2 (6)
3 (7)  4 (8)
```

Reply:
Oh, got it:
pressing the button  when the preset is already selected selects index+4

I'd say the description can be improved.

Reply:
> I get what you want to illustrate with that table, but it's not clear if I can press 1, then 2 to get effect 6?

No you can't, if you want effect 6 you need to press twice on button 2

> I'd say the description can be improved.

Yeah maybe the keyword should be "press twice to get second effect"